content/800-guides/240-partner-api.mdx

```diff
@@ -0,0 +1,163 @@
+
+- **Partner API**: A set of endpoints that allow approved partners to provision and manage Prisma Postgres databases on behalf of their users.
+- **Projects & Databases**: In Prisma Postgres, a "project" represents a database instance. When you provision a project, you are creating a new database for your user.
+- **Authentication**: All API requests require authentication. As a partner, you use OAuth2 to obtain integration tokens (for your app) and facilitate secure transfers to your users.
+- **Tokens**: There are two main types of tokens:
+  - **Integration Token**: Issued to your partner integration, used to provision and manage databases.
+  - **User Access Token**: Obtained via OAuth2 when a user authenticates, used to transfer database ownership to the user's workspace.
+
+## How to become a partner
+
+To use the Prisma Postgres Partner API, you first need to set up as a partner:
+
+1. **Request access to the Partner API**: Contact the Prisma team from the [Prisma Partners page](https://www.prisma.io/partners) to request access to the Partner API. You will be guided through the onboarding process.
+2. **Obtain OAuth credentials**: Once approved, you will receive an OAuth client ID and client secret. These credentials are required to authenticate your integration and enable secure database transfers for your users.
+
+For a complete list of available endpoints and details on request/response formats, see the [Prisma Management API documentation](/postgres/database/api-reference/management-api).
+
+## Provisioning a Database as a Partner
+
+To provision a new Prisma Postgres database for your users as a partner, follow these steps:
+
+1. **Collect user input**: Gather the necessary information from your user, such as the desired region, database name, and any other options your app supports.
+2. **Authenticate your integration**: Use your integration token (obtained via OAuth2) to authenticate API requests. This token proves your app is an approved partner.
+3. **Send a provisioning request**: Make a `POST` request to the Partner API endpoint to create a new project (database). For example:
+
+```ts
+const prismaResponse = await fetch('https://api.prisma.io/v1/projects', {
+  method: 'POST',
+  headers: {
+    'Content-Type': 'application/json',
+    Authorization: `Bearer <YOUR_INTEGRATION_TOKEN>`,
+  },
+  body: JSON.stringify({ region, name }),
+});
+```
+
+4. **Handle the response**: If successful, the API will return the new project's details, including connection strings and a `project_id`. Store these securely and display them to your user as needed.
+
+5. **(Optional) Store project metadata**: You may want to associate the `project_id` with your user in your own database for future reference.
+
+## Database Claim Flow
+
+Once a database is provisioned, you may want to transfer ownership to your user so they can manage it in their own Prisma workspace. This is done via the claim flow, which consists of three main steps:
```

Comment:
```suggestion
Once a database is provisioned, you may want to transfer ownership to your user at a later point so they can manage it in their own Prisma workspace and go beyond the free database usage limits. This is done via the claim flow, which consists of three main steps:
```

Something missing at the end? Mentions 3 steps, but then another section starts.